deploy/schemas/sample-backup-configs/sample_config_runtime.yaml

```diff
@@ -0,0 +1,47 @@
+run_farm:
+  base_recipe: str()
+  # map values unspecified since these are build farm specific
```

Comment:
I can look into this more. If instead of checking the schema based on a file, I could just pass yamale the dict given by `pyyaml` I think this should be easy to do (just run 2 verifications at the same time).

Reply:
Closing this to consolidate discussion.